app.js

```diff
@@ -34,7 +33,7 @@ if (process.env.V1_OAUTH_SECRETS_FROM_FILE) {
 }
 
 var app = express();
-var port = process.env.V1_OAUTH_SECRETS_FROM_FILE && url.parse(redirectUri).port || process.env.PORT;
+var port = (process.env.V1_OAUTH_SECRETS_FROM_FILE && process.env.PORT || 5000);
```

Comment:
I always ran on port 5000, but you can make this more configurable, or different, if you want.